index.html

```diff
@@ -36,35 +33,191 @@ <h1>
           me.
         </p>
         <h4>LET'S CONNECT</h4>
-        <section id="icons">
+        <div id="icons">
           <ul id="list-icons">
             <li class="img">
               <a href="#">
-                <img src="./image/tweeter.png" alt="">
+                <img src="./image/tweeter.png" alt="" />
               </a>
             </li>
             <li class="img">
               <a href="#">
-                <img src="./image/Linkedin icon.jpg" alt="">
+                <img src="./image/Linkedin icon.jpg" alt="" />
               </a>
             </li>
             <li class="img">
               <a href="#">
-                <img src="./image/m.png" alt="">
+                <img src="./image/m.png" alt="" />
               </a>
             </li>
             <li class="img">
               <a href="#">
-                <img src="./image/github.png" alt="">
+                <img src="./image/github.png" alt="" />
               </a>
             </li>
             <li class="img">
               <a href="#">
-                <img src="./image/hand.png" alt="">
+                <img src="./image/hand.png" alt="" />
               </a>
             </li>
           </ul>
-        </section>
+        </div>
+      </section>
+      <section id="section2">
+        <div id="tonic" class="work">
+          <img class="work-img" src="./image/image-work1.png" />
+          <h2 class="work-title">Tonic</h2>
+          <ul class="information">
+            <li>CANOPY</li>
+            <img src="./image/point.png" />
+            <li>Back End Dev</li>
+            <img src="./image/point.png" />
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button>See Project</button>

```

Comment:
- [ ] Please Implement the button interactions for all these buttons on the page 
![image](https://user-images.githubusercontent.com/97521557/235297788-9d136770-8456-43f7-95bc-2864cc258325.png)

Figma design 
![image](https://user-images.githubusercontent.com/97521557/235297807-46739b53-4473-471c-abc3-fd95c28ad9cf.png)